bin/apollo-schema-download.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 npm install -D apollo
-npx apollo schema:download --endpoint=https://staging.kickstarter.com/graph KsApi/graphql-schema.json --header "Authorization: token ${GRAPHQL_API_AUTH_TOKEN}"
+npx apollo schema:download --header="Content-Type: application/json" --header="User-Agent: Kickstarter iOS" --endpoint=https://staging.kickstarter.com/graph KsApi/graphql-schema.json
```

Comment:
Just a heads up that if we make changes to the user agent in the future (as some discussions have been occurring around this for our Segment implementation), we will probably have to update this too.

Reply:
Ah okay good to know.